Makefile

```diff
@@ -50,3 +50,6 @@ docker-build: test
 # Push the docker image
 docker-push:
 	docker push ${IMG}
+
+ginkgo:
+	ginkgo -v -race -randomizeAllSpecs ./pkg/... ./cmd/...
```

Comment:
Why not replace the `test` target with this? We could add a `ginkgo` target that checks `ginkgo` is installed before running the `test` target and then it should be safe for people

Reply:
I'm of the opinion that to run tests one should not have to install any external tools in go. The reason I added this was so that I could execute the tests in the same way as they are run in the Wercker container. Maybe there's a better name and/or I could add some comments to clarify the intention of this task?

Reply:
That's fair, let's leave it for now, I think the name is fine for now